Bugzilla/Template.pm

```diff
@@ -199,6 +199,13 @@ sub quoteUrls {
                ("\x{FDD2}" . ($count-1) . "\x{FDD3}")
               ~egox;
 
+    my $urlbase = Bugzilla->params->{'urlbase'};
```

Comment:
You should either move this prior to 195 above and replace the 195 use of `Bugzilla->params->{'urlbase'}`, or drop it and use `B->p->{'ub'}` same as the above 195.

Reply:
Good point